htlcswitch/interceptable_switch.go

```diff
@@ -139,11 +139,16 @@ func (f *interceptedForward) Resume() error {
 
 // Fail forward a failed packet to the switch.
 func (f *interceptedForward) Fail() error {
-	update, err := f.htlcSwitch.cfg.FetchLastChannelUpdate(
-		f.packet.incomingChanID,
-	)
-	if err != nil {
-		return err
+	update := f.htlcSwitch.failAliasUpdate(f.packet.incomingChanID, true)
```

Comment:
I think this PR is slightly behind master, as now it's possible for them to send a custom failure response here? So then it'd be the interceptor that needs to handle this leak, but I think this also makes sense here contextually as defense in depth.

Reply:
I think the user can leak their confirmed SCID here if they provide the failure response rather than just using a code